docs/markdown/FAQ.md

```diff
@@ -683,3 +683,21 @@ executable(
   link_language : 'c',
 )
 ```
+
+## How do I use `include-what-you-use` with meson?
+
+Following the steps from
+[Using with a compilation database](https://github.com/include-what-you-use/include-what-you-use#using-with-a-compilation-database),
+configure the build directory (e.g., with `meson setup build`), and run:
+
+```console
+$ iwyu-tool -p build
```

Comment:
Should be iwyu_tool, right?

Reply:
Hmm. I think on ubuntu the binaries are iwyu and iwyu_tool

Reply:
oh you're right https://packages.ubuntu.com/bionic/amd64/iwyu/filelist